### PR TITLE
Final (hopefully) editing

### PR DESCRIPTION
**Pull request by ketch**
Don't merge yet.  We'll discuss this PR today.

### CODE REVIEW

**Comment by ketch** [comment]
Okay, this is ready to merge now.

File: paper/eff_RH.tex [hunk 1]
```diff
@@ -357,66 +357,67 @@ \subsection{$C$-dispersion: effective dispersion induced by diffraction}\label{s
...
+However, one quickly finds that this is not the case.
+For example, in Figure \ref{fig:right_going_shock_theta45_intro} we consider
+propagation along $\theta=45^{\circ}$ in a medium with the same material properties as before.
+Again, we indicate the shock position predicted by \eqref{speed_of_shocks_z_dispersion} with
 a cyan dashed line. It is clear that the results in \cite{Ketcheson_LeVeque_2011}
 do not apply for the more general framework that we study in this work.
 %
+In Figure \ref{fig:right_going_shock_theta45_intro} we show
+with a dashed red line the position predicted by the theory developed in the present work.
 We propose an effective speed of propagation that is based on a leading order homogenized system.
```
Comment (manuel-quezada) [inline paper/eff_RH.tex]: Did you forget to comment out this line as well? (just making sure that you intended this line to be in the manuscript).

File: paper/eff_RH.tex [hunk 2]
```diff
@@ -197,15 +200,14 @@ \subsection{Viscous versus dispersive shocks}
...
 of propagation, one may observe shock formation (top left) dispersive shock formation
 (followed by the creation of solitary waves; top right) or more complicated behavior
 that lies between these extremes.
-For these particular cases, the material properties are the same but the angle of propagation changes. 
 
-It is also possible to study this problem via the modulation theory of \cite{whitham1965non} if
-one derives a high-order correction to the homogenized system. This is true since such
-correction is expected to be a dispersive term \cite{santosa1991}.
+It is also possible to study this problem via the modulation theory of \cite{whitham1965non},
+which leads to a high-order dispersive correction to the homogenized system \cite{santosa1991}.
```
[the remark this answers is not in the document]
Comment (ketch) [reply paper/eff_RH.tex]: You're right.

I think perhaps we should just remove this, however, as it's pretty speculative and we aren't planning to actually do it.

File: paper/eff_RH.tex (hunk 1 above)
Comment (ketch) [reply paper/eff_RH.tex]: This line is intended to be there.

I didn't comment out any lines.  The lines that show as deleted here just got moved to the next paragraph.

**Comment by ketch** [comment]
I've pushed additional changes that I think resolve the points you've raised.  Thanks for the careful review.

**Pull request merged**